pyproject.toml

```diff
@@ -214,7 +214,7 @@ type_incorrect = "Incorrect changelog type (see `changelog/README.rst`)"
 type_incorrect_long = "The filename of the changelog entry must be of the form `changelog/NUMBER.TYPE.rst` where `TYPE` is one of `feature`, `trivial`, `doc`, `bugfix`, `breaking`, or `removal`.\n\nFor details, see: https://docs.plasmapy.org/en/latest/contributing/changelog_guide.html#adding-a-changelog-entry"
 
```

Comment:
The background colour of the text here. Is it significant or just denoting something?

Reply:
I guess it's how GitHub now renders TOML files now?